### PR TITLE
school data edit from and some fixes

### LINKED ISSUE #31: Edit School form
Create form, and related functionalities for editing school details on the behalf of wise habit

### PR DESCRIPTION
**Pull request by saikrishnancnair**
#31 
Edit form for school is created and done some fixes with textarea.

### CODE REVIEW

File: app/Http/Controllers/Wisehabit/SchoolController.php [hunk 1]
```diff
@@ -52,6 +62,6 @@ public function createSchool(SchoolRequest $request)
     public function updateSchool($id, SchoolRequest $request)
     {
         $this->schoolService->update($id, $request, ['optional_values_if_any' => 'value']);
```
Comment (rahulreghunath) [inline app/Http/Controllers/Wisehabit/SchoolController.php:64]: Don't need to add optional_value_if any in all request. That is just to info there is an option to pass extra parameters.😅😂

**Review by rahulreghunath — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Wisehabit/SchoolController.php (hunk 1 above)
Comment (saikrishnancnair) [reply app/Http/Controllers/Wisehabit/SchoolController.php]: Corrected

Comment (rahulreghunath) [reply app/Http/Controllers/Wisehabit/SchoolController.php]: good

**Comment by rahulreghunath** [comment]
added digits validation rule instead min in mobile number validation

**Review by rahulreghunath — APPROVED** [review]

**Pull request merged**